insertion.py

```diff
@@ -0,0 +1,38 @@
+import time
+import random
+import matplotlib.pyplot as pyplot
+
+entries = []
+time_per_entry = []
+
+for i in range(10):
+    array = [random.randint(0, 100) for j in range(2_000*(i+1))]
+    start: float = time.time()
+    for j in range(len(array)):
+        insertIndex = j
+        for k in range(j):
+            if array[insertIndex] < array[k]:
+                insertIndex = k
+                break

```

Comment:
Tu reproduis le comportement d'un while avec for / if / break, je te conseille donc d'utiliser while